carbonmark-api/src/routes/activities/get.schema.ts

```diff
@@ -0,0 +1,35 @@
+import { Type } from "@sinclair/typebox";
+import { ActivityModel } from "../../models/Activity.model";
+import { NetworkParamModel } from "../../models/NetworkParam.model";
+
+export const QueryString = Type.Object({
+  network: Type.Optional(NetworkParamModel),
+  projectId: Type.Optional(
+    Type.String({
+      description:
+        "Desired projectID of the of the project the activities are related to",
+    })
+  ),
+  activityType: Type.Optional(
+    Type.String({
```

Comment:
Same as for IDs this should be `Type.Array(Type.String({...`

Reply:
Same as above